geoapps/drivers/components/factories.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, params: Params):
         self.inversion_type = params.inversion_type
         from SimPEG import dask
 
-        if self.inversion_type == "mvi":
+        if self.inversion_type in ["mvi", "magnetic"]:
```

Comment:
Can we mirror the naming of the classes? Maybe flags: "magnetic vector", "magnetic scalar"?

Reply:
Will be handled on #GEOPY-290